tests/network-tests/src/flows/proposals/vetoProposal.ts

```diff
@@ -36,7 +37,7 @@ export default async function vetoProposal({ api, query, lock }: FlowProps): Pro
       asMember: memberId,
       title: 'Veto proposal',
       description: 'Test veto proposal',
-    },
+    } as ProposalCreationParams<'VetoProposal'>,
```

Comment:
Avoid type assertions

Reply:
Addressed in [b552fba](https://github.com/Joystream/joystream/pull/3997/commits/b552fba6a40a75129fb358c4b771150313077472)